src/card/card-actions.jsx

```diff
@@ -16,9 +16,24 @@ function getStyles() {
 const CardActions = React.createClass({
 
   propTypes: {
+    /**
+     * Whether a click on this card component expands the card.
+     */
     actAsExpander: React.PropTypes.bool,
```

Comment:
Could you check the wording of the bool props? 'Replace in path' feature in WebStorm will help here.